python/cuml/tests/test_umap.py

```diff
@@ -674,26 +674,26 @@ def test_fuzzy_simplicial_set(n_rows, n_features, n_neighbors):
 
 
 @pytest.mark.parametrize(
-    "metric",
+    "metric,supported",
     [
-        "l2",
-        "euclidean",
-        "sqeuclidean",
-        "l1",
-        "manhattan",
-        "minkowski",
-        "chebyshev",
-        "cosine",
-        "correlation",
-        "jaccard",
-        "hamming",
-        "canberra",
+        ("l2", True),
```

Comment:
We should only have to do the mappings once- if you use strings in `SPARSE_SUPPORTED_METRICS` and `DENSE_SUPPORTED_METRICS` then you can literally just use the union of the two here here instead of having to list them out at all.

Reply:
Pulling in those lists into the test code would be counter-productive IMO since it correlates implementation and test expectation which means that it becomes harder to detect breaking changes.